web/src/main/scala/net/wiringbits/components/widgets/ResendVerifyEmailForm.scala

```diff
@@ -57,8 +64,10 @@ import scala.util.{Failure, Success}
           .sendEmailVerificationToken(request)
           .onComplete {
             case Success(_) =>
+              val email = formData.data.email.inputValue
+
               setFormData(_.submitted)
-              history.push("/verify-email")
+              history.push(s"/verify-email?email=${email}")
```

Comment:
This seems unnecessary

Reply:
I did that because the user can resend a second verify email after resend a first verify email, so the user doesn't need to re-type the email